src/module/cache-container/config/schemas/SchemaModalCtrl.ts

```diff
@@ -3,16 +3,16 @@ import {ISchemaDefinition} from "../../../../services/schemas/ISchemaDefinition"
 import {SchemaService} from "../../../../services/schemas/SchemasService";
 import {stringEndsWith} from "../../../../common/utils/Utils";
 import IModalServiceInstance = angular.ui.bootstrap.IModalServiceInstance;
```

Comment:
This is now unused, see `gulp check`

Reply:
Yes, we are definitely doing something wrong here. We should not remove the resource on the failed update.